Makefile

```diff
@@ -83,10 +84,11 @@ deploy: # Deploy/redeploy previously built REANA cluster.
 	source ${HOME}/.virtualenvs/${VENV_NAME}/bin/activate && \
 	eval $$(minikube docker-env --profile ${MINIKUBE_PROFILE}) && \
 	reana-cluster -f ${PWD}/../reana-cluster/reana_cluster/configurations/reana-cluster-latest.yaml down && \
+	if [ $$(helm ls reana-traefik | grep -q reana-traefik) ]; then helm delete --purge reana-traefik; fi && \
```

Comment:
It wasn't working? [`reana-cluster down` should do it](https://github.com/reanahub/reana-cluster/blob/master/reana_cluster/backends/kubernetes/k8s.py#L628) if traefik exists.

Reply:
I see, it could be that my cluster had `helm init` but `traefik was not deployed so it was failing....

I can delete it from here (so that Makefile would be nicer) but check whether the logic add if-helm-ls-then-helm-delete shouldn't be added to `down()` to cover for above scenarios...